packages/shared/src/components/profile/TopReaderWidget.tsx

```diff
@@ -78,13 +79,19 @@ export const TopReaderWidget = ({
           {topReaders.map((badge) => {
             return (
               <div className="flex justify-between" key={`badge-${badge.id}`}>
-                <Typography
-                  type={TypographyType.Caption1}
-                  color={TypographyColor.Primary}
-                  className="rounded-6 border border-border-subtlest-tertiary px-2.5 py-0.5"
+                <Link
+                  href={`${webappUrl}/tags/${badge.keyword.value}`}
+                  passHref
                 >
-                  {badge.keyword.flags.title}
-                </Typography>
+                  <Typography
+                    tag={TypographyTag.Link}
+                    type={TypographyType.Caption1}
+                    color={TypographyColor.Primary}
+                    className="rounded-6 border border-border-subtlest-tertiary px-2.5 py-0.5 transition duration-200 hover:bg-background-popover"
+                  >
+                    {badge.keyword.flags.title}
+                  </Typography>
```

Comment:
Do we not have a `Tag` component that does the redirection on its automatically?

Reply:
I did not find anything that matched the design